build/build-pr-ci.yml

```diff
@@ -63,9 +63,13 @@ steps:
       dotnet build $(System.DefaultWorkingDirectory)/src/WebJobs.Extensions.OpenAI/WebJobs.Extensions.OpenAI.csproj --configuration $(config) -p:WebJobsVersion=$(fakeWebJobsPackageVersion) -p:Version=$(fakeWebJobsPackageVersion)
       sudo cp "$(System.DefaultWorkingDirectory)/src/WebJobs.Extensions.OpenAI/bin/Release/Microsoft.Azure.WebJobs.Extensions.OpenAI.$(fakeWebJobsPackageVersion).nupkg" ~/NuGetPackagesLocal
 
+      sleep 10s
+      
       dotnet nuget add source ~/NuGetPackagesLocal
       dotnet nuget list source
 
+      sleep 10s
```

Comment:
This test is a bit flakey in that it can't always find the package in the local feed. I added these steps to see if that would help it, but it still seems to be a bit flakey :(